packages/babel-plugin-import-jsx-pragma/test/index.js

```diff
@@ -10,7 +10,7 @@ import plugin from '../src';
 
 describe( 'babel-plugin-import-jsx-pragma', () => {
 	function getTransformedCode( source, options = {} ) {
-		const { code } = transform( source, {
+		const { code } = transformSync( source, {
 			plugins: [
 				[ plugin, options ],
 				'syntax-jsx',
```

Comment:
I think this needs to be updated to `@babel/syntax-jsx` ? I'm seeing some related failing tests.

Reply:
yes